download-kubespray-files.sh

```diff
@@ -108,6 +108,49 @@ mkdir -p $FILES_DIR
 cp ${KUBESPRAY_DIR}/contrib/offline/temp/files.list $FILES_DIR/
 cp ${KUBESPRAY_DIR}/contrib/offline/temp/images.list $IMAGES_DIR/
 
+# --- Add Cilium Helm chart to files.list (derive version) ---
```

Comment:
Can you split this cilium helm chart specific part to other script in scripts directory and source from here?